.travis.yml

```diff
@@ -72,6 +72,8 @@ script:
     - bin/phpspec run --no-interaction -f dot
     - bin/phpunit
 
+    - bin/kawaii gherkin:check --align=left features/
```

Comment:
It would be more consistent to have this check before the tests and just after the code CS checker.